modules/ROOT/pages/execution-engine.adoc

```diff
@@ -16,11 +16,11 @@ execution, and the engine makes tuning decisions based on that metadata.
 Mule Event processors indicate to the runtime what kind of work they do, which 
 can be one of:
 
-* CPU Light: For quick operations, or Non-Blocking IO, for example, a Logger (`logger`) or HTTP Request operation (`http:request`).
-* Blocking IO: For IO that blocks the calling thread, for example, a Database Select operation (`db:select`).
-* CPU Intensive: For CPU-bound computations, for example, through the Transform Message component (`ee:transform`).
+* CPU Light: For quick operations, or Non-Blocking IO, for example, a Logger (`logger`) or HTTP Request operation (`http:request`). The applicable string in the logs is `CPU_LIGHT`.
```

Comment:
The applicable string in the logs is `CPU_LIGHT`. -> The applicable strings in the logs are `CPU_LITE` and `CPU_LITE_ASYNC`.

Reply:
@elrodro83 I incorporated your feedback.